keyboards/lily58/keymaps/nazu/keymap.c

```diff
@@ -0,0 +1,269 @@
+  XXXXXXX, XXXXXXX, KC_UP,   XXXXXXX, XXXXXXX, XXXXXXX,                     KC_DOT,  KC_P7,   KC_P8,   KC_P9,   KC_PSLS, KC_PAST,
+  _______, KC_LEFT, KC_DOWN, KC_RGHT, XXXXXXX, XXXXXXX,                     KC_COMM, KC_P4,   KC_P5,   KC_P6,   KC_PPLS, KC_PMNS,
+  _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,  XXXXXXX, XXXXXXX,  KC_P0,   KC_P1,   KC_P2,   KC_P3,   XXXXXXX, _______,
+                             _______, _______, _______,  _______, _______,  _______, _______, _______
+),
+
+/* ADJUST
+ * ,-----------------------------------------.                    ,-----------------------------------------.
+ * |RESET |      |      |      |      |      |                    |RGB TG| HUE+ | SAT+ | VAL+ |      |      |
+ * |------+------+------+------+------+------|                    |------+------+------+------+------+------|
+ * |      |      |QWERTY|      |COLEMK|      |                    | MODE | HUE- | SAT- | VAL- |      |      |
+ * |------+------+------+------+------+------|                    |------+------+------+------+------+------|
+ * |      |      |      |      |      |      |-------.    ,-------|      | MUTE |VOLDO | VOLUP|      |      |
+ * |------+------+------+------+------+------|       |    |       |------+------+------+------+------+------|
+ * |      |      |      |      |      |      |-------|    |-------|      | PREV |PLAY  | NEXT |      |      |
+ * `-----------------------------------------/      /      \      \-----------------------------------------'
+ *                   | LAlt | LGUI |LOWER | /Space /        \ Enter\ |RAISE |BackSP| RAlt |
+ *                   |      |      |      |/      /          \      \|      |      |      |
+ *                   `--------------------'------´            `------'--------------------´
+ */
+  [_ADJUST] = LAYOUT(
+  QK_BOOT, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                    RGB_TOG, RGB_HUI, RGB_SAI, RGB_VAI, XXXXXXX, XXXXXXX,
+  XXXXXXX, XXXXXXX, QWERTY,  XXXXXXX, CLMKDH,  XXXXXXX,                    RGB_MOD, RGB_HUD, RGB_SAD, RGB_VAD, XXXXXXX, XXXXXXX,
+  XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                    XXXXXXX, KC_MUTE, KC_VOLD, KC_VOLU, XXXXXXX, XXXXXXX,
+  XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,  KC_SWAP, KC_NORM, XXXXXXX, KC_MPRV, KC_MPLY, KC_MNXT, XXXXXXX, XXXXXXX,
+                               _______, _______,    _______,  _______, _______, _______, _______, _______
+  )
+};
+
+layer_state_t layer_state_set_user(layer_state_t state) {
+    state = update_tri_layer_state(state, _RAISE, _LOWER, _ADJUST);
+    return state;
+}
+
+//SSD1306 OLED update loop, make sure to enable OLED_ENABLE=yes in rules.mk
+#ifdef OLED_ENABLE
+
+oled_rotation_t oled_init_user(oled_rotation_t rotation) { return OLED_ROTATION_270; }
+
+bool oled_task_user(void) {
+    /* KEYBOARD PET VARIABLES START */
+
+    current_wpm   = get_current_wpm();
+    led_usb_state = host_keyboard_led_state();
```

Comment:
Neither of these appear to be used.

Reply:
They should be used in users/nazu/oled/oled_luna.c

Reply:
Well, in both cases, it's reading a variable back, so saving it locally shouldn't be necessary.